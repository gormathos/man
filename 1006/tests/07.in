C1;C2;C3;C4;C5;C6;C7;C8;C9;C10;C11;C12;C13;C14;C15;C16;C17;C18;C19;C20;C21;C22;C23;C24;C25;C26;C27;C28;C29;C30;C31;C32;C33;C34;C35;C36;C37;C38;C39;C40;C41;C42;C43;C44;C45;C46;C47;C48;C49;C50;C51;C52;C53;C54;C55;C56;C57;C58;C59;C60;C61;C62;C63;C64;C65;C66;C67;C68;C69;C70;C71;C72;C73;C74;C75;C76;C77;C78;C79;C80;C81;C82;C83;C84;C85;C86;C87;C88;C89;C90;C91;C92;C93;C94;C95;C96;C97;C98;C99;C100
20;54;25;32;18;43;54;13;20;21;21;69;47;50;26;61;52;35;52;24;68;49;69;54;77;39;76;52;19;57;58;11;79;20;18;40;60;28;53;38;38;35;55;69;80;33;20;14;27;36;42;65;52;67;57;56;59;36;72;69;68;66;75;81;28;19;73;14;11;47;73;71;15;55;61;41;22;59;22;60;20;36;43;27;29;64;50;67;33;12;45;63;77;59;77;67;13;56;76;49
21;68;40;66;43;39;75;78;51;68;39;35;78;23;51;49;72;34;13;11;53;33;20;51;26;18;51;40;31;31;17;72;29;71;46;76;44;50;70;24;46;15;12;22;24;79;50;77;76;28;70;62;17;55;65;58;74;58;15;62;72;72;21;30;73;35;79;67;79;16;37;25;19;30;72;31;62;27;46;34;22;18;38;31;57;33;77;10;47;43;48;28;70;70;80;28;14;78;62;44
21;68;63;40;42;16;17;20;19;15;41;39;66;66;38;68;43;76;45;74;36;70;25;66;14;48;46;81;50;22;50;61;35;24;23;70;16;41;66;41;44;22;55;28;12;70;51;74;62;64;68;27;13;16;43;73;24;27;72;32;36;60;34;28;59;81;62;30;77;14;36;59;60;14;18;73;20;25;57;76;52;15;57;16;30;40;53;79;71;71;51;20;63;30;22;61;81;13;62;75
73;25;33;81;49;36;53;62;25;53;54;29;46;24;67;47;21;22;25;79;11;28;75;17;69;50;41;78;16;58;76;46;74;43;50;54;70;60;16;13;11;22;27;13;30;57;11;59;25;62;44;28;40;14;18;52;53;13;49;76;66;33;54;49;69;52;67;57;13;13;53;23;38;52;47;80;80;36;55;63;13;48;76;78;71;33;43;30;80;63;80;80;31;41;17;62;54;15;18;16
66;51;71;58;39;14;75;65;21;15;27;35;67;32;74;42;34;59;44;19;57;22;70;81;33;68;23;65;25;31;17;20;19;26;74;23;26;23;70;34;16;48;50;43;49;60;23;58;57;25;35;37;56;55;31;38;60;64;47;72;71;73;15;46;38;50;40;45;66;15;67;80;69;53;31;21;49;43;11;15;57;13;73;44;59;28;80;28;42;26;13;44;48;72;37;28;45;25;77;21
70;44;17;15;32;27;60;23;42;17;24;46;32;60;45;35;80;69;67;29;34;12;59;38;61;54;55;71;64;16;25;15;48;32;16;38;34;64;34;76;67;11;38;72;70;71;60;31;75;52;48;78;14;78;77;66;79;81;69;31;17;36;25;21;67;77;56;20;77;81;73;77;38;78;75;56;32;79;50;11;50;72;43;66;12;47;17;24;10;76;64;74;40;39;43;26;37;21;48;70
11;54;50;76;74;31;54;46;36;55;34;23;10;27;42;21;41;20;15;56;63;56;45;72;80;78;65;38;12;28;19;77;11;30;54;31;63;60;52;33;14;22;77;25;53;79;76;75;54;10;40;25;59;76;19;31;46;68;45;37;55;70;60;80;68;29;22;18;37;80;13;24;57;51;45;62;68;67;64;26;65;22;39;43;24;36;43;61;43;52;30;10;10;66;38;31;24;19;61;67
31;61;78;38;40;47;32;29;43;48;63;63;47;14;63;58;73;51;53;72;45;12;60;73;36;54;77;67;35;56;80;43;64;21;20;42;40;13;76;77;51;15;81;76;22;77;22;72;33;31;77;72;49;22;71;43;11;34;51;63;40;78;80;75;63;33;49;55;53;46;23;45;25;73;63;50;24;65;43;79;40;59;72;30;28;26;21;67;22;11;15;47;10;61;59;32;26;73;71;65
53;70;35;38;11;81;26;41;64;52;80;68;79;74;36;61;80;53;68;81;18;49;54;56;27;13;41;26;37;73;48;72;55;34;40;20;68;60;79;67;74;59;35;44;14;39;62;20;53;69;70;39;76;64;76;24;61;58;80;50;67;58;73;67;57;39;79;77;53;31;45;37;38;73;57;26;44;77;78;25;64;52;37;69;62;50;45;51;73;41;41;65;16;69;45;56;14;75;40;51
32;78;37;15;64;56;51;77;22;74;76;69;59;77;30;22;60;10;11;55;18;19;31;16;55;79;55;23;51;26;31;77;46;78;32;62;54;26;65;55;62;43;78;78;14;46;56;23;59;43;35;42;13;65;30;45;68;18;17;70;42;25;30;78;67;71;24;56;73;39;40;70;72;59;31;28;11;40;63;54;51;64;60;54;12;68;18;55;70;62;16;35;26;66;15;42;19;19;28;64
72;70;44;17;36;33;67;66;60;57;75;68;67;45;58;72;72;12;73;77;28;65;57;20;27;23;35;71;10;72;72;68;59;13;61;71;62;58;65;34;55;69;29;39;53;63;31;34;70;52;71;37;59;39;72;34;79;14;19;39;22;33;47;50;56;43;71;18;75;42;39;34;53;58;19;28;14;32;80;63;52;61;62;12;73;23;13;28;28;68;18;18;10;62;80;74;43;10;38;74
28;26;38;48;22;44;66;65;27;68;45;61;38;58;61;20;19;20;69;74;46;80;22;27;40;19;39;28;28;50;45;51;34;27;69;18;50;81;32;35;70;10;57;25;70;61;41;44;66;53;76;38;11;52;14;56;51;48;65;30;21;52;64;45;36;15;19;68;17;68;40;26;69;79;79;72;77;52;54;13;12;74;55;71;59;51;61;73;23;78;77;22;66;75;65;19;15;55;11;69
62;62;73;31;53;56;61;10;70;78;74;42;47;50;69;69;39;47;49;51;56;42;77;45;42;58;19;80;39;27;72;48;34;19;36;34;23;67;56;11;11;30;40;48;32;69;70;50;51;31;14;63;45;26;11;41;65;41;50;76;55;30;52;46;18;80;33;57;73;64;37;11;51;73;61;56;66;80;79;63;74;27;81;39;16;65;23;23;34;14;51;58;18;76;65;21;59;77;77;61
48;64;75;26;34;68;31;15;44;41;47;19;13;80;53;45;52;22;79;42;27;71;27;71;29;45;17;53;58;10;32;10;19;53;41;36;53;47;64;40;40;25;19;54;47;65;50;65;11;19;55;11;32;59;14;38;73;42;32;48;46;32;67;57;23;10;22;54;45;75;66;32;81;29;25;64;81;54;23;74;32;37;58;60;55;36;17;59;61;41;62;19;68;60;55;16;80;34;23;14
46;53;23;33;79;60;75;63;33;30;45;22;62;43;49;67;32;39;56;80;35;80;66;42;21;69;79;35;40;42;60;54;80;61;30;69;12;30;70;24;11;38;12;78;26;17;55;37;21;45;53;24;44;46;18;60;68;20;11;57;45;44;46;48;52;80;28;48;65;21;25;54;29;68;25;42;32;81;81;13;49;79;63;74;42;24;42;10;64;63;69;60;31;26;25;38;81;10;51;43
54;33;33;17;49;14;66;28;42;46;52;27;58;40;70;19;69;57;52;16;75;38;51;62;67;67;15;29;55;66;77;47;61;39;22;81;55;60;13;39;65;28;66;31;28;14;22;37;68;31;26;49;28;51;30;73;65;57;14;32;73;48;64;65;23;45;11;34;27;27;34;62;50;27;57;13;68;32;35;72;44;23;17;68;37;56;27;34;46;38;29;43;68;51;72;65;78;14;68;73
59;75;62;61;30;34;70;69;54;66;26;50;55;27;61;69;60;40;37;26;60;52;66;39;23;79;65;77;16;38;35;47;73;20;34;56;21;25;55;15;58;13;70;75;10;76;72;28;49;16;34;74;68;39;37;34;28;51;63;14;47;63;80;73;74;65;41;15;27;67;50;59;26;38;15;32;13;76;48;64;45;33;60;71;75;73;49;34;46;48;33;79;78;22;47;65;81;16;71;28
22;47;53;10;43;49;74;30;47;55;76;78;28;18;13;80;10;52;80;34;16;51;40;21;17;69;38;15;75;54;61;59;12;68;55;67;46;52;62;53;26;54;28;21;70;23;53;77;14;74;65;38;76;35;31;51;45;71;69;39;44;34;26;28;45;53;39;44;58;38;22;50;10;57;53;62;70;58;25;10;57;61;63;23;79;52;66;34;38;36;67;48;66;79;69;31;65;69;27;65
14;62;29;14;34;29;52;70;18;40;74;63;40;62;80;68;75;57;54;47;45;49;74;57;70;33;16;73;47;64;27;47;12;65;69;73;16;12;64;67;35;22;78;12;75;36;74;44;39;28;76;21;28;44;40;26;16;19;77;28;81;59;65;44;65;21;64;54;59;35;56;50;50;64;81;17;43;22;34;61;37;40;25;14;28;30;14;55;16;27;75;17;43;66;43;62;13;23;30;28
21;36;37;58;43;35;66;22;12;21;78;64;36;29;76;26;68;35;63;76;30;44;51;66;17;39;12;53;74;51;38;42;29;79;12;29;22;33;45;26;54;22;50;30;37;55;37;26;25;19;55;66;52;69;32;53;28;54;50;12;14;66;81;48;63;72;27;16;65;14;53;81;74;74;48;51;40;40;21;27;36;31;71;22;28;35;10;52;42;67;20;16;25;45;47;22;40;29;73;21
20;18;42;52;61;56;64;38;53;36;56;25;59;36;54;14;65;43;34;73;23;32;63;59;41;30;60;31;68;10;18;81;46;78;47;51;78;11;65;74;65;57;16;29;37;23;25;55;66;35;12;38;65;46;35;76;67;22;32;75;66;53;39;76;35;70;42;16;49;77;16;40;30;55;53;69;33;32;18;40;27;71;59;33;71;46;34;69;65;56;38;29;36;50;81;63;35;39;80;53
11;31;65;68;10;12;51;77;26;40;76;28;37;72;67;14;46;33;16;26;11;45;27;26;77;78;14;55;73;69;17;26;71;69;62;52;75;17;44;47;73;45;58;76;27;68;58;36;11;34;59;16;35;38;75;53;58;71;66;53;46;22;30;38;30;54;47;17;53;54;72;37;64;32;47;16;43;24;42;41;12;19;79;58;44;42;45;15;41;53;27;36;21;73;80;62;10;59;76;52
65;28;38;20;60;13;72;14;68;31;71;53;38;25;50;49;46;23;45;22;76;38;34;12;72;21;14;65;72;20;33;57;45;52;12;33;80;71;27;19;16;45;44;11;70;48;78;41;63;30;58;13;60;40;44;17;61;13;14;33;32;62;48;10;30;58;55;49;41;61;54;28;60;20;52;59;19;38;52;78;63;61;15;18;37;47;15;43;22;36;43;20;24;69;41;59;40;39;41;66
71;14;22;24;14;17;24;41;45;68;45;80;42;69;43;48;43;15;73;11;58;36;14;29;45;54;52;51;67;65;50;18;75;54;76;56;53;15;46;52;73;33;44;35;20;24;65;34;41;26;75;67;45;71;27;63;55;42;59;59;71;34;38;39;53;37;71;45;75;60;52;45;57;33;75;65;61;76;24;16;65;48;46;35;58;31;70;43;23;40;54;17;70;61;62;16;81;66;47;44
10;55;33;43;19;47;73;19;64;57;22;25;49;24;25;68;56;72;53;12;78;14;72;67;34;52;47;36;50;32;76;36;79;10;67;24;43;80;69;42;54;31;64;23;34;23;60;69;70;80;54;20;64;27;64;51;53;49;17;21;75;44;39;55;77;40;62;13;80;59;19;60;61;38;78;11;22;39;71;31;23;72;66;70;66;45;25;25;49;48;66;56;32;74;37;23;21;41;58;31
48;79;19;77;68;67;20;21;30;26;29;80;36;15;10;77;71;73;42;49;11;26;14;47;18;61;81;64;67;54;26;69;69;18;75;79;73;65;46;73;10;10;46;21;55;71;54;73;18;56;10;47;21;70;56;61;49;77;60;50;30;80;46;45;49;68;60;81;47;71;70;31;64;46;74;49;42;36;61;23;77;77;25;53;78;68;38;72;77;52;24;22;73;43;66;44;57;27;24;41
42;45;34;31;15;44;78;28;10;49;58;62;51;76;81;12;20;35;76;69;23;56;39;17;55;41;78;30;59;24;74;54;39;39;33;69;21;61;43;35;59;71;13;71;45;61;69;44;47;21;60;16;38;18;20;61;20;30;18;47;62;14;34;43;75;71;51;40;62;72;52;78;37;74;63;11;74;69;11;72;21;44;50;26;70;35;22;15;24;32;74;64;53;47;53;33;13;11;42;45
51;22;61;78;75;24;54;81;13;23;13;66;35;43;33;33;47;15;48;67;37;32;12;18;66;40;18;77;27;45;79;37;67;81;26;67;69;47;34;78;17;21;69;44;28;66;21;39;62;36;49;56;16;53;22;65;72;62;43;43;10;70;46;37;66;61;57;76;46;51;29;69;33;72;65;62;81;57;22;46;28;69;76;61;49;42;30;58;71;10;79;24;22;41;51;71;26;75;50;74
36;45;52;48;70;47;11;38;42;77;57;38;60;24;39;67;16;41;12;51;30;12;60;69;71;78;55;60;57;39;34;22;17;78;11;78;67;20;70;51;62;74;36;49;21;63;42;66;25;14;54;52;59;39;76;63;78;53;64;53;63;77;52;73;66;77;45;78;49;22;60;41;21;34;46;60;52;51;56;19;20;70;27;24;75;19;39;54;14;81;49;45;44;65;60;34;24;73;31;16
29;36;48;10;13;37;73;71;62;56;47;37;65;14;49;50;11;74;37;30;43;39;63;61;51;38;17;69;23;51;79;55;56;34;62;58;66;25;53;27;73;13;78;42;15;52;25;60;24;76;38;81;76;53;45;67;64;62;44;21;80;45;74;81;77;72;66;29;40;37;21;74;37;28;19;75;40;44;36;65;28;25;73;43;22;36;39;68;33;29;52;31;33;58;61;29;17;48;66;47
46;56;79;76;17;10;51;15;20;47;11;19;54;38;63;13;30;47;38;45;60;26;15;13;63;64;43;70;55;16;69;22;47;54;21;63;25;69;42;28;67;53;56;35;80;29;29;62;44;77;62;78;79;73;67;11;69;20;34;61;62;31;79;57;56;80;59;41;63;63;37;34;57;23;27;14;60;27;77;74;52;16;81;78;72;66;29;49;37;61;76;16;76;52;12;20;31;69;42;57
44;18;56;43;44;24;19;13;45;32;60;54;14;76;52;66;46;24;53;51;45;10;38;80;14;34;57;55;18;17;52;59;76;18;55;80;35;75;32;34;80;54;24;52;37;81;52;64;52;66;55;31;57;57;36;29;26;11;66;36;56;15;23;12;19;62;25;20;41;12;43;80;26;10;77;11;42;54;13;72;21;21;53;69;69;51;62;18;11;70;78;27;34;23;10;65;19;54;11;48
23;37;65;72;11;23;68;37;65;47;65;44;46;57;32;66;62;17;15;27;70;48;35;24;54;62;80;79;41;56;23;51;31;13;40;58;32;47;81;63;26;49;50;56;46;52;22;67;28;33;39;62;57;73;16;66;49;50;65;33;59;13;73;63;76;36;55;47;75;67;10;59;62;43;68;56;56;19;18;67;46;17;13;76;46;48;13;66;49;45;70;27;48;42;59;71;51;73;42;70
73;79;49;78;31;31;64;21;66;19;65;63;32;63;50;15;34;22;54;22;21;34;39;50;70;58;19;55;54;33;35;37;23;43;37;21;42;44;56;22;23;67;23;16;32;77;39;25;14;57;81;64;56;72;81;29;24;56;24;66;19;23;19;24;54;70;46;33;33;81;37;66;63;68;59;49;28;16;33;52;45;48;58;50;32;71;35;57;79;40;20;54;35;79;63;67;35;37;14;55
48;33;76;30;74;19;30;64;32;69;55;40;46;73;53;80;22;28;34;72;60;22;56;65;52;17;26;55;33;69;54;81;41;57;77;74;79;51;45;76;39;26;60;33;63;46;30;60;66;16;32;43;81;60;23;51;81;34;20;73;22;34;10;55;78;43;65;75;80;64;13;68;32;44;30;61;40;37;11;43;33;54;80;74;43;64;31;66;53;27;61;67;28;19;74;10;50;76;46;25
37;71;43;72;75;55;49;36;41;54;45;41;51;79;75;28;18;52;15;29;41;29;71;42;35;45;19;49;35;58;48;53;44;24;22;42;15;39;53;61;54;54;45;40;26;60;69;56;54;25;33;73;79;64;41;23;54;11;12;53;45;19;56;53;33;48;50;46;39;38;32;69;45;38;32;15;64;53;49;29;13;20;65;62;67;73;10;62;29;29;58;18;40;52;54;76;62;31;73;42
73;22;44;56;11;42;36;56;33;81;80;30;67;62;78;67;53;72;17;40;48;76;74;36;21;76;52;42;35;65;50;48;78;39;17;29;31;51;35;37;76;28;27;71;54;63;73;47;31;33;65;11;75;31;60;78;68;32;67;21;41;57;10;30;63;59;66;36;20;59;43;28;60;20;19;78;16;52;65;62;11;79;41;49;13;24;59;16;67;16;66;48;48;65;65;42;20;70;18;37
52;77;17;27;73;61;56;49;36;58;16;18;68;50;19;29;34;14;62;39;33;18;80;13;66;55;65;18;79;64;81;24;27;16;79;11;61;78;71;17;44;44;33;13;25;20;81;79;53;68;15;29;34;73;47;59;15;21;23;58;51;49;67;50;30;57;29;61;47;28;75;35;28;33;16;79;63;78;18;16;26;80;42;59;24;58;49;33;51;44;57;56;81;55;32;66;32;35;21;52
61;42;27;24;38;30;50;54;74;72;46;17;37;35;75;54;40;65;78;19;21;78;58;71;71;57;51;46;38;34;52;44;69;76;20;41;40;31;30;80;62;35;52;46;46;52;10;14;75;58;36;22;27;14;69;14;17;37;62;62;49;36;38;55;35;39;80;20;57;10;79;67;35;77;65;72;10;57;17;26;57;45;71;33;65;25;61;32;41;26;76;30;14;45;53;29;11;69;60;53
50;75;30;14;36;16;24;14;47;79;80;57;10;29;57;18;13;33;12;60;30;48;70;69;27;26;61;38;27;23;30;51;31;41;36;44;38;52;64;26;32;73;31;45;75;53;22;29;32;37;45;58;10;75;49;46;64;42;48;34;34;59;58;59;16;44;39;31;76;79;49;63;14;67;55;60;77;71;18;76;51;14;63;80;28;43;77;64;28;32;15;76;42;22;78;45;57;45;81;17
52;59;13;52;43;60;37;74;51;69;62;31;58;67;69;57;71;34;74;68;40;16;68;70;79;20;41;72;15;77;72;62;28;78;26;48;32;46;28;54;57;42;60;15;72;53;22;66;62;77;38;39;73;78;11;19;66;21;69;32;41;43;16;43;70;39;12;79;26;72;31;18;25;33;27;70;27;70;18;43;22;61;71;15;54;29;75;17;20;77;46;51;38;30;73;30;56;68;48;63
48;33;38;15;75;71;48;19;31;80;67;43;20;10;64;40;55;44;34;59;55;76;22;11;22;33;76;79;11;31;53;41;59;65;50;42;46;57;53;66;14;11;33;61;29;55;68;47;71;55;13;79;58;27;23;46;56;31;80;13;66;18;14;17;44;54;12;57;81;48;60;25;22;12;52;14;41;25;35;37;51;56;25;67;34;49;58;75;45;31;60;45;10;26;68;32;64;70;77;34
16;20;81;24;76;78;18;26;12;46;37;66;52;44;26;27;31;79;21;69;24;55;35;49;42;41;12;21;64;31;63;71;64;55;56;64;15;48;28;32;58;55;65;34;28;12;52;70;57;32;20;27;61;13;75;18;59;11;31;17;41;70;64;30;50;80;53;41;24;67;62;43;31;10;42;60;44;59;68;55;10;12;32;43;66;10;29;43;46;39;69;30;61;54;32;73;79;56;74;48
65;19;65;74;33;13;66;16;48;33;39;58;16;40;40;32;49;67;56;64;47;51;79;18;81;66;44;70;59;44;60;69;52;79;54;23;66;23;17;33;15;70;24;44;45;28;72;20;40;34;32;44;45;62;24;54;56;40;29;68;32;28;16;26;33;24;64;16;37;24;33;20;69;25;18;46;10;67;73;63;14;19;20;40;28;26;24;63;14;46;19;12;10;72;64;66;34;19;11;48
12;18;69;46;60;46;57;23;32;22;77;60;29;72;73;50;53;64;64;26;42;30;21;28;74;73;81;53;61;68;36;80;44;24;35;65;30;80;27;71;46;81;19;27;35;28;62;23;81;70;21;15;28;42;70;46;17;28;65;78;16;53;28;11;46;48;32;16;45;32;48;27;65;36;14;19;22;77;65;75;52;35;15;51;17;19;66;62;67;60;58;68;75;16;24;43;54;50;25;58
