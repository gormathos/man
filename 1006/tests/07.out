207423
81
10
